mycroft_bus_client/message.py

```diff
@@ -91,16 +108,18 @@ def reply(self, msg_type, data=None, context=None):
         Returns:
             Message: Message object to be used on the reply to the message
         """
-        data = data or {}
+        data = deepcopy(data) or {}
         context = context or {}
 
-        new_context = self.context
+        new_context = deepcopy(self.context)
```

Comment:
We need to add `from copy import deepcopy` at the top of the file for this to work

Reply:
Missed that; strange that it didn't throw any exceptions when I tested this branch.. Fixed now.